File: python/ifelifelse/desafio2.py
    # 2) Escreva um programa que leia os valores A, B, C e imprima na tela se a soma de A +
    #    B é menor que C.

a = input("Digite um número: ")
b = input("Digite mais um número: ")
c = input("Digite um terçeiro número: ")

if(a+b < c):
    print("A soma do primeiro número "+a+" mais a soma do segundo número "+b+" é menor que o número "+c)
elif(a+b > c):
    print("A soma do primeiro número "+a+" mais a soma do segundo número "+b+" é maior que o número "+c)
else:
    print("A soma do primeiro número "+a+" mais a soma do segundo número "+b+" é igual ao número "+c)

File: java/ifelseifelse/desafio2.java
package java.ifelseifelse;

import java.util.Scanner;

/*
     *@autor: Hylan Silva
     *@version: 0.0.5 
     */
public class desafio2 {
    // 2) Escrva um programa que leia os valores A, B, C e imprima na tela se a soma de A +
    //    B é menor que C.
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a,b,c;

        System.out.print("Digite o primeiro número: ");
        a = in.nextInt();
        System.out.print("Digite o segundo número: ");
        b = in.nextInt();
        System.out.print("Digite o terceiro número: ");
        c = in.nextInt();
        in.close();

        if(a+b < c){
            System.out.println("a soma do primeiro valor: "+a+" junto com a soma do sengudo valor: "+b+" é menor que o terceiro valor: "+c);
        }else if(a+b > c){

        }else{

        }
    }
}
